modules/nodes-pods-vertical-autoscaler-about.adoc

```diff
@@ -40,3 +40,14 @@ Administrators can use the VPA to better use cluster resources, such as preventi
 ====
 If you stop running the VPA or delete a specific VPA CR in your cluster, the resource requests for the pods already modified by the VPA do not change. However, any new pods get the resources defined in the workload object, not the previous recommendations made by the VPA.
 ====
+
+[id="nodes-pods-vertical-autoscaler-modes_{context}"]
+== About the Vertical Pod Autoscaler Operator modes
+
+You can use a `VerticalPodAutoscaler` CR to specify the mode that the VPA operates in:
+
+* The `InPlaceOrRecreate` mode automatically applies the VPA CPU and memory recommendations throughout the pod lifetime. When any pod in the project is out of alignment with the VPA recommendations, the VPA attempts to apply updates _in-place_, without restarting any of the containers in the pod. If the VPA is not able to update the containers in-place, the VPA deletes the pod. When redeployed by the workload object, the VPA updates the new pod with its recommendations. For specific information, see "About the in place or recreate mode". 
```

Comment:
> without restarting any of the containers in the pod.

I think it should be without `without restarting the pod`.

We should also mention that the [resizePolicy on the container](https://kubernetes.io/docs/tasks/configure-pod-container/resize-container-resources/#container-resize-policies) dictates the way in which in-place resizes will be applied. If `NotRequired`, then the container will not be restarted, best effort. If `RestartContainer`, then in-place updates will always restart the container.

Reply:
@maxcao13 

>without restarting any of the containers in the pod.

I am  a bit unclear on which component gets restarted (and the docs seem a bit confused, too). I was thinking after we get this feature documented, perhaps we can do another pass to clean this up.